### PR TITLE
Add Spaces to Task Data

### PR DESCRIPTION
**Pull request by beurkinger**
required for https://github.com/huggingface/moon-landing/pull/4861

This PR add a "spaces" attribute to the TaskData, used for displaying a Spaces section on the Task Page.

Note: I run the linted on the config files, so it may be better to check "Hide whitespace" in the "Files changed" tab :)

### CODE REVIEW

File: tasks/src/audio-classification/data.ts [hunk 1]
```diff
@@ -58,6 +58,7 @@ const taskData: TaskDataCustom = {
 			id:          "superb/hubert-large-superb-er",
 		},
 	],
+	spaces:  [],
```
Comment (merveenoyan) [inline tasks/src/audio-classification/data.ts:61]: What are the keys for the dict in this list? description & id?

**Review by merveenoyan — APPROVED** [review]
LGTM, @osanseviero do you have an objection?

**Review by osanseviero — APPROVED** [review]
LGTM, thanks! No objections

**Comment by merveenoyan** [comment]
maybe we could merge after the one on 🌜🛩 is merged 😅

**Comment by osanseviero** [comment]
Actually this one will need to be merged before https://github.com/huggingface/moon-landing/pull/4861, as the internal one depends on this

File: tasks/src/audio-classification/data.ts (hunk 1 above)
Comment (julien-c) [reply tasks/src/audio-classification/data.ts:61]: it's an `ExampleRepo` same as the other repo types

**Review by julien-c — COMMENTED** [review]
i haven't followed too much, the actual data will be filled in some subsequent PRs?

**Comment by osanseviero** [comment]
Yes, Merve is curating a list of spaces, which is shared in Slack in the task pages channel

**Comment by merveenoyan** [comment]
@julien-c PR is here #589

**Pull request merged**